reference/data/models/basal/deliverytype.v1.yaml

```diff
@@ -1,6 +1,6 @@
 title: Delivery Type
 description: >-
-  This defines the basal delivery type, used to differentiate between [automated](./automated.v1.yaml), [scheduled](./scheduled.v1.yaml), [suspended](./suspend.v1.yaml) and [temporary](./temporary.v1.yaml) basal types.
+  This defines the basal delivery type, used to differentiate between [automated](../automated.v1.yaml), [scheduled](../scheduled.v1.yaml), [suspended](#schema/suspend.v1) and [temporary](../temporary.v1.yaml) basal types.
```

Comment:
The links without the `#schema` prefix seem to link to the raw OpenAPI spec whereas the one with `#schema` seems to be pretty much a no-op (opens the same page?)

Was that intentional?

Reply:
We may want to drop the links like you did with `bolus.v1.yaml`

Reply:
Ah, this is what I wanted to show you - how it just opens the raw spec when using `../`. I forgot to drop the links after our call, so thanks for pointing this out!